src/main/java/com/puppycrawl/tools/checkstyle/checks/naming/CatchParameterNameCheck.java

```diff
@@ -35,16 +35,16 @@
  * <li>allows {@code e} abbreviation (suitable for exceptions end errors)</li>
  * <li>allows {@code ex} abbreviation (suitable for exceptions)</li>
  * <li>allows {@code t} abbreviation (suitable for throwables)</li>
+ * <li>allows {@code _} for unnamed catch parameter</li>
```

Comment:
```suggestion
 * <li>allows {@code _} for unnamed catch parameters</li>
```
Other items are plural, let's be consistent

Reply:
done